requirements/base.in

```diff
@@ -2,14 +2,14 @@
 
 -c ../constraints.txt
 
-boto==2.39.0
-cookiecutter==0.9.0
-Django>=1.11
-django-pyfs==2.0
-fs-s3fs==0.1.8
-lxml==3.8.0
-lazy==1.1
-requests==2.22.0
+boto
+cookiecutter
+Django>=2.2
```

Comment:
This file is not being used by setup.pt, so I guess that it is Ok to move this constraint to constraints.txt